src/ontology/mondo-edit.obo

```diff
@@ -525685,16 +525787,19 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 
 [Term]
 id: MONDO:0957408
-name: type 1 interferonopathy of childhood
+name: type 1 interferonopathy
```

Comment:
we actually had an Orphanet term that was labeled type 1 interferonopathy (MONDO:0018782) that we excluded as part of our mass obsoletion.

Should we:
- point to this term MONDO:0957408 type 1 interferonopathy as the replaced by for the obsolete term?
- add the Orphanet ID as an xref on this term (MONDO:0957408)?
    - if we do that though, we should reconsider the classification in ORDO and if those terms should also be children of new term (actually, we should review ORDO's hierarchy regardless and determine if those terms should be grouped under this term (MONDO:0957408)
- _or_ we could keep this childhood term and reinstate the obsoleted MONDO:0018782 'obsolete type 1 interferonopathy'?